core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -501,7 +501,7 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
         val args = { env map { _ ++ action.parameters } getOrElse action.parameters } merge payload
         val message = Message(
             transid,
-            FullyQualifiedEntityName(action.namespace, action.name, action.version),
+            FullyQualifiedEntityName(action.namespace, action.name, Some(action.version)),
```

Comment:
Since FullyQualifiedEntityName has a singleton, you could have added a second apply method which accepts SemVer and boxes it into Some instead.